docs/source/reference/modding/extended.rst

```diff
@@ -0,0 +1,208 @@
+
+This feature can be used to implement a street illumination, glowing windows, etc.
+
+Advantages of described approach over old wods with glowing windows:
+
+1. There is no need to spawn additional objects for day and night mode
+
+2. There is no need to attach a script to every switchable object
+
+An example of mod which uses this feature is the `Glow in the Dahrk`_.
+
+
+Per-group animation files support
+---------------------------------
+
+In original engine it is possible to add a custom animation file to NPC to override some animations (usually idle ones).
+In OpenMW it is possible to override animations via the same file for all actors which use a given basic animation file.
+
+If you want to override animations for all bipedal actors (they use the xbase_anim.nif skeleton), you can put your animations to the
+``Animations/xbase_anim`` folder in your ``Data Files`` (you can use the "multiple data folders" feature here).
+In this case any bipedal actor without custom animation will use your animations, but if he has additional animations, they have the higher priority.
+
+For example, all bipedal actors by default use the same spellcasting animations, so the xbase_anim override is enough.
+If you want to override walking animations, you should override the ``xbase_anim_female`` and ``xbase_anim_kna`` for females and beast races as well,
+because they have own walking animations, so they override ones which come from the xbase_anim (including its overrides from ``Animations`` folder).
+
+To enable this feature, you should have this line in your settings.cfg:
+
+``[Game]``
+
+``use additional anim sources = true``
+
+An example of mod which uses this feature is the `Almalexia's Cast for Beasts`_.
+
+
+Weapon sheathing support
+------------------------
+
+In OpenMW it is possible to show sheathing weapons on actors back, including quivers and scabbards.
+
+This feature conflicts with old mods which use a scripted scabbards, arrows with particles or decorative quivers (attached to the left pauldron, for example).
+
+1. Basics
+
+The minimum you need is the ``xbase_anim_sh.nif`` file from the `Weapon Sheathing`_ mod and this line in your settings.cfg:
+
+``[Game]``
+
+``weapon sheathing = true``
+
+The ``xbase_anim_sh.nif`` contains default placement points for different weapon types.
+It is enough to get the Gothic-style weapon sheathing for all bipedal actors (without quivers and scabbards).
+
+2. Scabbards
+
+To show a scabbard, you need a mesh with ``_sh`` suffix. For example, if a weapon uses the ``foo.sh`` mesh, a sheat mesh should have the ``foo_sh.nif`` name.
+There should be an least two nodes in the sheath file:
+
+``Bip01 Weapon`` - represents a weapon itself (may be just a grip for sword, for example). Is not shown when the weapon is drawn.
+
+``Bip01 Sheath`` - represents scabbards, quivers, etc. It is shown always when the weapon is equipped.
```

Comment:
```suggestion
``Bip01 Sheath`` - represents scabbards, quivers, etc. It is shown always, even when the weapon is equipped.
```

Reply:
The term "equipped" means "equipped in slot", not "drawn".